resources/metadata/shellExecute.yaml

```diff
@@ -0,0 +1,32 @@
+metadata:
+  name: shellExecute
+  description: Step executes defined script
+  longDescription: Step executes defined script with Vault credentials, or created them on this step
+spec:
+  inputs:
+    params:
+      - name: vaultServerUrl
+        type: string
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        description: The URL for the Vault server to use
+        mandatory: true
```

Comment:
we should not make this mandatory.

Can't we get rid of the Vault parameters overall here?
It's part of the default functionality, isn't it?

If keeping it here again it may clash with the default functionality?

Reply:
removed